docs/install.rst

```diff
@@ -0,0 +1,352 @@
+.. highlight:: bash
+
+.. _install:
+
+==================
+Quick installation
+==================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Preface
+=======
+
+This part of the documentation covers the installation of CrateDB on Linux,
+macOS and Windows systems.
+The first step to using any software package is getting it properly installed.
+Please read this section carefully.
+
+
+Introduction
+============
+
+By installing CrateDB the way as outlined in this document, you will be able to
+quickly setup and run a single-node cluster.
+
+After the installation is finished, the CrateDB service should be up and
+running. Now, you can access the :ref:`Admin UI <admin-ui:index>` from your
+local machine by navigating to::
+
+    http://localhost:4200/
+
+After successfully installing the software, you might want to follow up with
+:ref:`taking the guided tour <use>`.
+
+Also, you might enjoy to be guided into further information about how to
+quickly adjust the :ref:`configuration settings <configure>`, read more
+details about the :ref:`reference:config` of CrateDB, the background about
+:ref:`howtos:bootstrap-checks`, multi-node configuration within the section
+about :ref:`howtos:clustering` and :ref:`howtos:going-into-production`. When
+operating a CrateDB cluster in production, `performance tuning`_ will also be
+of interest.
+
+.. _performance tuning: https://crate.io/docs/crate/howtos/en/latest/performance/
+
+.. NOTE::
+
+    To add additional CrateDB nodes to this kind of cluster in order to make it
+    form a multi-node cluster, you will need to remove the cluster state after
+    changing the configuration.
+
+.. NOTE::
+
+    If you are interested to follow the development more closely, please check
+    out the :ref:`reference:release_notes` for more specific information about
+    each CrateDB release. `Other releases of CrateDB`_ are also available.
+
+
+Prerequisites
+=============
+
+CrateDB requires a `Java virtual machine`_ to run.
+
+- Starting with CrateDB 4.2, a `Java virtual machine`_ is bundled with CrateDB
+  and no extra installation is necessary.
+
+- CrateDB version 3.0 to 4.1 required a separate Java installation, version
+  11 was the minimum requirement. Earlier versions required Java 8.
+  We recommend using OpenJDK_ on Linux Systems and `Oracle Java`_ on
+  Microsoft Windows and macOS.
```

Comment:
```suggestion
- CrateDB versions before 4.2 required a separate Java installation. For CrateDB 3.0 to 4.1, Java 11 is the minimum requirement. CrateDB versions before 3.0 require Java 8.
  We recommend using OpenJDK_ on Linux Systems and `Oracle Java`_ on
  Microsoft Windows and macOS.
```

Otherwise the first part of the sentence implies that versions before 3.0 required no separate Java installation - that is than contradicted by the second sentence.